pkg/networkservice/common/mechanisms/sendfd/client.go

```diff
@@ -40,7 +40,7 @@ func NewClient() networkservice.NetworkServiceClient {
 func (s *sendFDClient) Request(ctx context.Context, request *networkservice.NetworkServiceRequest, opts ...grpc.CallOption) (*networkservice.Connection, error) {
 	// Get the grpcfd.FDSender
 	rpcCredentials := grpcfd.PerRPCCredentials(grpcfd.PerRPCCredentialsFromCallOptions(opts...))
-	opts = append(opts, grpc.PerRPCCredentials(rpcCredentials))
+	opts = append([]grpc.CallOption{grpc.PerRPCCredentials(rpcCredentials)}, opts...)
```

Comment:
Why do we need to change the order of call options?

Reply:
My fault, I did a mistake during the testing, there is no need for this.